### DISCUSSION TITLE
Unable to download power pages site. Error: Website record [Website ID] doesn't exist.

### DISCUSSION THREAD

**Discussion by cfelek** [microsoft/powerplatform-build-tools#675] [Q&A] [answered]
In Visual Studio Code, I authenticated and I successfully got a list of sites and their Website ID's. 
I get an error when trying to download the files in a VSC terminal. I tried a few of the Website ID's, disabled VPN, enabled javascript, enabled third party cookies, disabled cookie blocking, flushed cash, did all upgrades, rebooted and I got the same error.
The auth profile and environment is visible in VSC.

Would you be able to help?

------------------------------------------------------------------------------

<img width="203" alt="VSC download powerpages" src="https://github.com/microsoft/powerplatform-build-tools/assets/117937066/290d8009-757c-4e7b-a1a8-46016c5ba3b3">

------------------------------------------------------------------------------

pac paportal download --path "[path]" --webSiteId [Website ID]
also tried...
pac powerpages download --path "[path]" --webSiteId [Website ID]

Information:
Be careful when you're updating public sites. The changes you make are visible to anyone immediately. To check site visibility go to https://make.powerpages.microsoft.com/.
To get additional information, please visit https://go.microsoft.com/fwlink/?linkid=2204350

Started downloading website [Website ID]
Connected to... [Server Name]
Connected as [Username]@[Email Domain]
Downloading: Website [adx_website]...
Microsoft PowerPlatform CLI
Version: 1.29.11+g9e2b163
Online documentation: https://aka.ms/PowerPlatformCLI
Feedback, Suggestions, Issues: https://github.com/microsoft/powerplatform-build-tools/discussions

Error: Website record [Website ID] doesn't exist.

**Comment by cfelek** [comment, marked as answer]
Solution: 

https://learn.microsoft.com/en-us/power-pages/admin/enhanced-data-model#data-model-power-platform-cli-parameters

When you use the Power Platform CLI to upload or download configuration data for a website that uses the enhanced data model, you must use the **modelVersion parameter. A value of 2** indicates that the enhanced data model should be used.

pac powerpages download --path <path> --webSiteId <siteId> --modelVersion 2

**Reply by africauniv** [reply]
Thank you very much, I was making a CI CD pipeline for a powerpages project and for half a day I couldn't make the pac powerpages download.

**Reply by crptbddst** [reply]
legend
